config/initializers/mandate.rb

```diff
@@ -22,6 +53,15 @@ def active_job_queue
     end
 
     def defer(*args, wait: nil, **kwargs)
+      if kwargs[:prereq_jobs]
```

Comment:
```suggestion
      # We need to convert the jobs to a hash before we serialize as there's not
      # serialization format for a job. We do this here to avoid cluttering the codebase
      # with this logic.
      if kwargs[:prereq_jobs]
```